manifest.config.ts

```diff
@@ -46,24 +38,31 @@ export const common: Manifest.WebExtensionManifest = {
 		page: 'src/ui/options/index.html',
 		open_in_tab: true,
 	},
+
+	icons: {
+		16: 'icons/icon_generic_16.png',
```

Comment:
Why does this have a different name?

Reply:
It uses a different icon. This was the case in v2 as well, where there is a generic icon (which is just red and white) for 16x16 due to it being so small. I dont know where this icon actually is used but I kept it to be safe